pwndbg/commands/nearpc.py

```diff
@@ -97,6 +99,10 @@ def nearpc(pc=None, lines=None, to_string=False, emulate=False):
 
         line   = ' '.join((prefix, "%#x" % i.address, s or '', asm))
 
+        # Highlight the current line if the config is enabled
```

Comment:
Well it just highlights stuff in green when it does not have explicit color.  All constants etc will preserve it's color, did you try running the patch?